planning/scenario_planning/lane_driving/behavior_planning/lane_change_planner/include/lane_change_planner/data_manager.hpp

```diff
@@ -15,42 +16,40 @@
 #ifndef LANE_CHANGE_PLANNER__DATA_MANAGER_HPP_
 #define LANE_CHANGE_PLANNER__DATA_MANAGER_HPP_
 
-#include <memory>
-
-// Own package
-#include "lane_change_planner/parameters.hpp"
+// ROS
+#include <geometry_msgs/msg/pose_stamped.hpp>
```

Comment:
Replace <> with "" except for C or C++ header.

Reply:
I think this should not be changed. It does not affect the functionality, but it has a bad effect on the performance because it searches the current directory unnecessary.

Reply:
OK. We decided to use double quotes except for system headers.

Reply:
Fixed.